stdlib/REPL/src/latex_symbols.jl

```diff
@@ -416,8 +416,8 @@ const latex_symbols = Dict(
     "\\lq" => "‘",
     "\\rq" => "’",
     "\\reapos" => "‛",
-    "\\quotedblleft" => "“",
-    "\\quotedblright" => "”",
```

Comment:
Any reason not to keep the old aliases as well?

Reply:
In the past, when we simplified/rationalized the tab completions, we got rid of the old completions.  Leaving them in complicates things like the REPL help (e.g. if someone types `?“` to find out the tab completion) unnecessarily.

I doubt that anyone was using the awkward old completions, since up to now these characters can't be used in Julia syntax (except within strings).

Reply:
Ok, sounds fine to me then